package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "gulp": "gulp",
+    "pages": "npm run gulp",
```

Comment:
@brockfanning fixed it. remembered you need to have a `pages` command to build anything beforehand.